src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -7348,8 +7348,33 @@ void MusicXMLParserPass2::harmony(const String& partId, Measure* measure, const
             ha->setRootTpc(Tpc::TPC_INVALID);
             ha->setBaseTpc(Tpc::TPC_INVALID);
             functionText = m_e.readText();
-            // TODO: parse to decide between ROMAN and NASHVILLE
             ha->setHarmonyType(HarmonyType::ROMAN);
+        } else if (m_e.name() == "numeral") {
+            ha->setRootTpc(Tpc::TPC_INVALID);
+            ha->setBaseTpc(Tpc::TPC_INVALID);
+            while (m_e.readNextStartElement()) {
+                if (m_e.name() == "numeral-root") {
+                    String numeralRoot = m_e.readText();
+                    String numeralRootText = m_e.attribute("text");
+                    // TODO analyze text and import as roman numerals
+                    ha->setHarmonyType(HarmonyType::NASHVILLE);
```

Comment:
That is not entirely true, because the `numeral` element has been ignored until here, the old behavior for `function` hasn't changed. But I will add a basic check for Roman numerals in the text attribute.